.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,6 +3,13 @@ description: Report your bugs with the theme here!
 labels: [bug]
 body:
   - type: checkboxes
+    attributes:
+      label: Something is actually broken, I am not requesting help with my configuration.
+      description: Please open a discussion if you need support.
```

Comment:
We should probably specify what type of discussion and directly link to it.
You can see my [prior art](https://github.com/catppuccin/jetbrains/blob/270eaff65e91e4914df6840ebb2b6dc7512d5465/.github/ISSUE_TEMPLATE/bug.yml?plain=1#L15) where I figured out that `description` tags allow markdown links.